modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,75 +25,79 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
 .Example output
 [source,terminal]
 ----
-NAME                       READY   STATUS    RESTARTS      AGE
-pod/ovnkube-master-9g7zt   6/6     Running   1 (48m ago)   57m
-pod/ovnkube-master-lqs4v   6/6     Running   0             57m
-pod/ovnkube-master-vxhtq   6/6     Running   0             57m
-pod/ovnkube-node-9k9kc     5/5     Running   0             57m
-pod/ovnkube-node-jg52r     5/5     Running   0             51m
-pod/ovnkube-node-k8wf7     5/5     Running   0             57m
-pod/ovnkube-node-tlwk6     5/5     Running   0             47m
-pod/ovnkube-node-xsvnk     5/5     Running   0             57m
+Warning: apps.openshift.io/v1 DeploymentConfig is deprecated in v4.14+, unavailable in v4.10000+
+NAME                                         READY   STATUS    RESTARTS       AGE
+pod/ovnkube-control-plane-65c6f55656-6d55h   2/2     Running   0              114m
+pod/ovnkube-control-plane-65c6f55656-fd7vw   2/2     Running   2 (104m ago)   114m
+pod/ovnkube-control-plane-65c6f55656-vtqtm   2/2     Running   0              114m
+pod/ovnkube-node-bcvts                       8/8     Running   0              113m
+pod/ovnkube-node-drgvv                       8/8     Running   0              113m
+pod/ovnkube-node-f2pxt                       8/8     Running   0              113m
+pod/ovnkube-node-frqsb                       8/8     Running   0              105m
+pod/ovnkube-node-lbxkk                       8/8     Running   0              105m
+pod/ovnkube-node-tt7bx                       8/8     Running   1 (102m ago)   105m
 
-NAME                            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)             AGE
-service/ovn-kubernetes-master   ClusterIP   None         <none>        9102/TCP            57m
-service/ovn-kubernetes-node     ClusterIP   None         <none>        9103/TCP,9105/TCP   57m
-service/ovnkube-db              ClusterIP   None         <none>        9641/TCP,9642/TCP   57m
+NAME                                   TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)             AGE
+service/ovn-kubernetes-control-plane   ClusterIP   None         <none>        9106/TCP            114m
```

Comment:
portr here is 9108 remember I pointed you to the CNO PR in our previous iteration? cc @ricky-rav